test/python/test_status_api.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+import unittest
+
+import openapi_client
+
+from . import api_config
+
+AUTHENTICATOR_FIELDS = [
+    "enabled",
+    "installed",
+    "configured",
+]
+
+class TestAuthenticatorsApi(api_config.ConfiguredTest):
+    """AuthenticatorsApi unit test stubs"""
+    def setUp(self):
+        self.api = openapi_client.api.status_api.StatusApi(self.client)
+
+    def test_authenticators(self):
+        """Test case for authenticators
+
+        """
+        authenticators = self.api.authenticators()
+
+        self.assertIsInstance(authenticators, openapi_client.models.authenticators.Authenticators)
+
+        for i in AUTHENTICATOR_FIELDS:
+            lst  = getattr(authenticators, i)
+            self.assertIsInstance(lst, list)
+            self.assertIn('authn', lst)
+            for value in lst:
+                self.assertIsInstance(value, str)
```

Comment:
Do we know what the output values should actually be? If it's something we can anticipate, we should assert on it to be that. 

i.e., we should assert that the key `enabled` has the value 'auth', etc.

Reply:
The only one we can really make any assumptions about is the `authn` authenticator. Which should be in all three lists. I check that with the `assertIn` statement

Reply:
Great, I think I missed that line, thank you for clarifying!